metrics/release.go

```diff
@@ -32,6 +32,22 @@ var (
 		},
 	)
 
+	ReleaseAttemptDeploymentSeconds = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "release_attempt_deployment_seconds",
+			Help:    "Release durations from the moment the SnapshotEnvironmentBinding was created til the release is marked as deployed",
+			Buckets: []float64{0.5, 1, 2, 3, 4, 5, 6, 7, 10, 15, 30},
```

Comment:
You sure about this buckets? They seem quite small tbh.

Reply:
No I have no idea what average deployment times are

Reply:
They seem unrealistic to be honest. @scoheb do you have some numbers for us?